apps/google-analytics-4/lambda/src/middlewares/setSentryContext.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('setSentryContext', () => {
   app.use(setSentryContext);
 
   app.get('/test', (_req, res) => {
-    res.send('ok');
+    res.sendStatus(200);
```

Comment:
is there a difference between `status(200)` and `sendStatus(200)`?

Reply:
deprecation issue